modules/tsb/cp/manifests/tsb/controlplane-values.yaml.tmpl

```
@@ -34,7 +38,9 @@ spec:
     xcp:
       centralAuthMode: JWT
       configProtection: {}
-      enableHttpMeshInternalIdentityPropagation: false
+      %{ if id_propagation_enabled }
+      enableHttpMeshInternalIdentityPropagation: true
+      %{ endif }
```

[nit], less verbose:
```suggestion
      enableHttpMeshInternalIdentityPropagation: ${id_propagation_enabled}
```

---

makes sense, overlooked at the start...